### ISSUE TITLE
Accessibility tab index

### ISSUE THREAD

**Issue by allisonjw** [trishalanglois/refactor-tractor#21]
The link below provides great information about adding the tabindex attribute. All elements on the page, that can be accessed by the tab on the keyboard, need the attribute. Most times it's tabindex="0" attribute, but the number value can change for different reasons. 

- https://webaim.org/techniques/keyboard/tabindex

**Comment by trishalanglois** [comment]
Upon further research, it seems like tabindex should be used for elements that the user will interact with, e.g. an input on a form.  At this point, it doesn't actually seem like we have anything that the user will interact with.  Because of that, I'm going to hold off on putting this in our HTML until we change it to include interactive elements, e.g. when we have an input form for the user to put their name/id number or when they're adding more data.  I'll take care of this when we get to that point, though.
